watertap/unit_models/tests/test_uv_aop.py

```diff
@@ -207,32 +218,60 @@ def test_solve(self, UV_frame):
     @pytest.mark.component
     def test_solution(self, UV_frame):
         m = UV_frame
-        assert pytest.approx(0.999999999, rel=1e-3) == value(
+        assert pytest.approx(2052.999848078, rel=1e-3) == value(
             m.fs.unit.control_volume.properties_in[0].flow_mass_phase_comp["Liq", "H2O"]
         )
-        assert pytest.approx(0.001003, rel=1e-3) == value(
+        assert pytest.approx(2.0592, rel=1e-3) == value(
             m.fs.unit.control_volume.properties_in[0].flow_vol
         )
-        assert pytest.approx(74e-9, rel=1e-3) == value(
+        assert pytest.approx(1.51922e-4, rel=1e-3) == value(
             m.fs.unit.control_volume.properties_in[0].flow_mass_phase_comp[
                 "Liq", "NDMA"
             ]
         )
-        assert pytest.approx(0.999999999, rel=1e-3) == value(
+        assert pytest.approx(2053, rel=1e-3) == value(
             m.fs.unit.control_volume.properties_out[0].flow_mass_phase_comp[
                 "Liq", "H2O"
             ]
         )
-        assert pytest.approx(1.8248e-8, rel=1e-3) == value(
+        assert pytest.approx(4.8104e-5, rel=1e-3) == value(
             m.fs.unit.control_volume.properties_out[0].flow_mass_phase_comp[
                 "Liq", "NDMA"
             ]
         )
-        assert pytest.approx(5000, rel=1e-3) == value(m.fs.unit.uv_dose)
+        assert pytest.approx(500, rel=1e-3) == value(
+            pyunits.convert(m.fs.unit.uv_dose, to_units=pyunits.mJ / pyunits.cm**2)
+        )
         assert pytest.approx(0.04576, rel=1e-3) == value(m.fs.unit.UVA)
-        assert pytest.approx(1.3333e-4, rel=1e-3) == value(
-            m.fs.unit.reaction_rate_constant["Liq", "NDMA"]
+        assert pytest.approx(2.3e-3, rel=1e-3) == value(
+            m.fs.unit.photolysis_rate_constant["Liq", "NDMA"]
+        )
+        assert pytest.approx(1156.99, rel=1e-3) == value(
+            pyunits.convert(m.fs.unit.electricity_demand[0], to_units=pyunits.kW)
+        )
+
+    @pytest.mark.requires_idaes_solver
+    @pytest.mark.component
+    def test_costing(self, UV_frame):
```

Comment:
@luohezhiming Based on @bknueven 's previous suggestion, I think it'd be good to have a quick test checking that max energy consumption gets selected as intended, so maybe a separate test with two target components (two EE/Os), and assert that the expected energy consumption is set to the actual energy consumption.

Reply:
Not sure if you did include that somewhere and I missed it...if not, do you think you can add that?

Reply:
I could try to add another dummy component to the test to validate the test.